packages/react-aria-components/docs/ComboBox.mdx

```diff
@@ -405,7 +405,7 @@ as shown below, can be used when the options come from an external data source s
 
 As seen below, an iterable list of options is passed to the ComboBox using the `defaultItems` prop. Each item accepts an `id` prop, which
 is passed to the `onSelectionChange` handler to identify the selected item. Alternatively, if the item objects contain an `id` property,
-as shown in the example below, then this is used automatically and an `id` prop is not required.
+as shown in the example below, then this is used automatically and a `key` prop is not required.
```

Comment:
Thanks, however, all the RAC  ones should be `id`, and not `key`. We moved away from key in the newer API's because it's a React special prop that we don't have access to.
See https://github.com/adobe/react-spectrum/discussions/7147 for more information

Reply:
Ah, I see. I misunderstood. Thank you!